src/gc/gccommon.cpp

```diff
@@ -151,6 +151,9 @@ InitializeGarbageCollector(
     /* Out */ GcDacVars* gcDacVars
     )
 {
+
+    STRESS_LOG0(LF_STARTUP, LL_ALWAYS, "===== InitializeGarbageCollector start \n");
```

Comment:
don't be confused with logging like that, just debugging artifacts while this is still WIP